pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -91,9 +91,10 @@ func (podDisruptionBudgetStrategy) ValidateUpdate(ctx api.Context, obj, old runt
 	return append(validationErrorList, updateErrorList...)
 }
 
-// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects.
+// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects. Status update should
```

Comment:
I wasn't talking about pods but about PDB. Even if AllowUnconditionalUpdate returns true, and the eviciton REST tries to update the status of a PDB with a rv older than what etcd has you will get an update conflict.